apps/vaporgui/PRegionSelector.cpp

```diff
@@ -0,0 +1,35 @@
+#include "PRegionSelector.h"
+#include "QRangeSliderTextCombo.h"
+#include "VLineItem.h"
+#include <vapor/ParamsBase.h>
+#include <QVBoxLayout>
+
+PRegionSelector::PRegionSelector(const std::string &tag, const std::string &label)
+: PWidget(tag, _container = new QWidget)
+{
+    QVBoxLayout *layout = new QVBoxLayout;
+    layout->setMargin(0);
+    _container->setLayout(layout);
+    
+    layout->addWidget(new VLineItem("X", _sx = new QRangeSliderTextCombo));
+    layout->addWidget(new VLineItem("Y", _sy = new QRangeSliderTextCombo));
+    layout->addWidget(new VLineItem("Z", _sz = new QRangeSliderTextCombo));
+    
+    _sx->setEnabled(false);
+    _sy->setEnabled(false);
+    _sz->setEnabled(false);
+    
+//    _vgeometry->SetRange(<#const std::vector<float> &range#>)
```

Comment:
Remove commented out code.

Reply:
This code is not finished because at the time of writing VGeometry selector was broken. I added code to prevent the class from being created but I need to revisit it to check if VGeometry has been fixed.

Reply:
VGeometry is still broken (I tried VGeometry2 which I assume is the current version)